.github/workflows/ds.yml

```diff
@@ -30,11 +30,13 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.sha }}
+      - uses: actions-rs/toolchain@v1
```

Comment:
Can't we just remove (not add) this whole `uses` statement?

Reply:
done